test/examples/setup_examples_ros2.launch.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+  <!-- Include rosbridge websocket launch file for ROS 2 -->
+  <include file="$(find-pkg-share rosbridge_server)/launch/rosbridge_websocket_launch.xml">
+    <arg name="call_services_in_new_thread" value="true"/>
```

Comment:
turns out our tests accidentally depend on this behavior which was not default in humble